etc/translations/README.md

```diff
@@ -2,14 +2,14 @@
 
 In order to help translating LibreMines, it is necessary to have the [Qt Linguistic Tool](https://doc.qt.io/qt-5/qtlinguist-index.html) installed.
 
-On Arch Linux and its derivatives, it can be done by installing rhe package `qt5-tools` (or `qt6-tools` for Qt6):
+On Arch Linux and its derivatives, it can be done by installing the package `qt6-tools` (or `qt5-tools` for Qt5):
 ```sh
-sudo pacman -S qt5-tools
+sudo pacman -Sy --needed qt6-tools
 ```
 
-On Ubuntu systems, it can be done with:
+On Debian and its derivatives, it can be done with:
 ```sh
-sudo apt install qttools5-dev
+sudo apt-get install qt6-tools-dev
```

Comment:
please replace Deabian based systems package for qt6-tools-dev-tools. It can be kept as apt instead of apt-get

Reply:
ok, it seems this will work. But include [linguist-qt6](https://packages.debian.org/bookworm/linguist-qt6) as well.